stock_vertical_lift/models/vertical_lift_operation_base.py

```diff
@@ -0,0 +1,502 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from collections import namedtuple
+
+from odoo import _, api, fields, models
+
+from odoo.addons.base_sparse_field.models.fields import Serialized
+
+_logger = logging.getLogger(__name__)
+
+
+# The following methods have been copied from 'shopfloor' module (OCA/wms)
+# TODO: we should move them in a generic module
+
+
+def split_other_move_lines(move, move_lines):
+    """Substract `move_lines` from `move.move_line_ids`, put the result
+    in a new move and returns it.
+    """
+    other_move_lines = move.move_line_ids - move_lines
+    if other_move_lines or move.state == "partially_available":
+        qty_to_split = move.product_uom_qty - sum(move_lines.mapped("product_uom_qty"))
+        backorder_move_vals = move._split(qty_to_split)
+        backorder_move = move.create(backorder_move_vals)
```

Comment:
You have to confirm the new move:
```suggestion
        backorder_move = move.create(backorder_move_vals)
        backorder_move._action_confirm(merge=False)
```
See https://github.com/OCA/wms/blob/b0ceac26303ef8278d641fb050399d3c965ecc6c/shopfloor/models/stock_move.py#L41-L43

Reply:
@hailangvn what about this?  Also, are we sure we always get values? We might have an empty dict AFAIR. Can you check?